gpu-usage

```diff
@@ -8,37 +8,47 @@ print_usage () {
 Usage: $0 [-n charles[01-19]]
```

Comment:
We could change this to `-n nodelist`. I'm not gonna ask for that, I actually think this is clearer!